### ISSUE TITLE
gnt-backup import expects wrong datatype

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#118]
Originally reported of Google Code with ID 62.

```
<b>What steps will reproduce the problem?</b>
1. Export an instance using following command:
gnt-backup export -d -n NODE_OF_INSTANCE INSTANCE_NAME

2. Import instance from Step 1:
gnt-backup import -d -n NODE_OF_INSTANCE --net 0:ip=IP_OF_INSTANCE,bridge=BRIDGE_OF_INSTANCE,mac="MAC_OF_INSTANCE" -s 
5G -B memory=512M,vcpus=1 -H xen-pvm:root_path=/dev/xvda1 -t plain INSTANCE_NAME

<b>What is the expected output? What do you see instead?</b>
gnt-backup import should import the exported instance. Import failes, because gnt-backup script expects a dictionary 
for opts.nics and calls items method on opts.nics in line 105 of gnt-backup, but opts.nics is a list.

<b>What version of the product are you using? On what operating system?</b>
Ganeti 2.0.2 and git version on CentOS 5.3
```
Originally added on 2009-08-03 14:00:40 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Thanks for catching this. Will be fixed in 2.0.3.
```
Originally added on 2009-08-04 12:32:59 +0000 UTC.

Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was fixed in commit dc922da and will be released with 2.0.3.

thanks,
iustin
```
Originally added on 2009-08-05 08:47:54 +0000 UTC.

Added Labels: Type-Defect
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
